core/validation/txValidator.go

```diff
@@ -96,7 +96,9 @@ func VerifyTransaction(Tx *tx.Transaction, ledger *ledger.Ledger, TxPool []*tx.T
 }
 
 func CheckMemPool(tx *tx.Transaction, TxPool []*tx.Transaction) error {
-
+	if len(tx.UTXOInputs)==0{
```

Comment:
"if len(tx.UTXOInputs)==0{"  should be follow the format  "if len(tx.UTXOInputs) == 0 {" which is the default "go fmt" style, it is better run "go fmt" before submit the code